packages/analytics-nextjs/src/context.tsx

```diff
@@ -42,6 +43,28 @@ export function useAnalyticsContext() {
     return analyticsContext;
 }
 
+function PlausibleWrapperMaybe({ newsroom, children }: PropsWithChildren<Pick<Props, 'newsroom'>>) {
+    if (!newsroom || !newsroom.is_plausible_enabled) {
+        return <>{children}</>;
+    }
+
+    return (
+        <PlausibleProvider
+            domain={`${newsroom.uuid}.prezly.com`}
```

Comment:
This should use the [new SDK property](https://github.com/prezly/javascript-sdk/commit/95954ed562b8fe0af7614784bca05502b3539794).